spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    email { "user@mail.com" }
+    password { "password" }
+    password_confirmation { "password" }
+    role {"user"}
+  end
+end
```

Comment:
I would like to see a factory for a journalist as well. Something like this will give you 2 factories that you can use:

```ruby
FactoryBot.define do
  factory :user do
    email { "user@mail.com" }
    password { "password" }
    password_confirmation { "password" }
    role { :user }
  end
  
  factory :journalist do
    role { :journalist }
  end
end
```

Also, please remember to format your code....

Reply:
ok

Reply:
address this